requirements.in

```diff
@@ -1 +1,2 @@
 Flask
+supabase
```

Comment:
oops, this was left in while i was testing something, but we will need this in the future anyways

Reply:
same with all the new reqs in requirements.txt

Reply:
nvm... this should be removed